amundsen_application/static/js/components/common/Bookmark/MyBookmarks/constants.ts

```diff
@@ -1,5 +1,4 @@
-export const ITEMS_PER_PAGE = 4;
-export const PAGINATION_PAGE_RANGE = 10;
-
 export const BOOKMARK_TITLE = "My Bookmarks";
+export const BOOKMARKS_PER_PAGE = 4;
 export const EMPTY_BOOKMARK_MESSAGE = "You don't have any bookmarks. Use the star icon to save a bookmark.";
+export const MY_BOOKMARKS_SOURCE_NAME = "Bookmarks";
```

Comment:
nit: I think we're currently using `snake_case` for these, so it would be `bookmarks`.